src/ProjectService.Data/UserRepository.cs

```diff
@@ -34,7 +35,7 @@ public void AddUsersToProject(IEnumerable<DbProjectUser> dbProjectUsers , Guid p
 
         public IEnumerable<DbProjectUser> Find(Guid userId)
         {
-            return _provider.ProjectsUsers.Where(x => x.UserId == userId);
+            return _provider.ProjectsUsers.Include(u => u.Project).Where(x => x.UserId == userId).ToList();
```

Comment:
but if we don't want to include the project? I think it is not good practice

Reply:
done